src/mpid/common/genq/mpidu_genqi_shmem_types.h

```diff
@@ -9,13 +9,43 @@
 #include "mpl.h"
 #include "mpidu_init_shm.h"
 
+typedef void *MPIDU_genq_shmem_pool_t;
+typedef void *MPIDU_genq_shmem_queue_t;
```

Comment:
The transparent queue handle should be exposed in the public mpidu_genq_shmem_queue.h.

Reply:
See above.